element_test.go

```diff
@@ -815,3 +815,18 @@ func (t T) ElementErrors() {
 	err = el.Context(ctx).Release()
 	t.Err(err)
 }
+
+func (t T) ElementMoveMouseOut() {
+	p := t.page.MustNavigate(t.srcFile("fixtures/click.html"))
+	btn := p.MustElement("button")
+	btn.MustEval(`this.onmouseenter = () => this.dataset['a'] = 1`)
```

Comment:
You should also add a check for [onmouseout](https://developer.mozilla.org/en-US/docs/Web/API/GlobalEventHandlers/onmouseout)